### DISCUSSION TITLE
FAO ECodes or GRIB users

### DISCUSSION THREAD

**Discussion by richardgilham** [MetOffice/simulation-systems#70] [UM]
Paul Dando from ECMWF is reaching out for GRIB users regarding a planned change. Please get in touch if this is relevant to you...

ECMWF plans to start outputting model data at sub-hourly forecast time steps soon and we'll need some changes in ecCodes so that we can archive this in MARS.  I want to check that changes we make aren't going to break any GRIB encoding done at the Met Office.

Ideally, it would be good to have some sample GRIBs produced by the Met Office encoded with sub-hourly forecast time steps as well the code that's used to produce this.  As I assume the main output from the UM is still PP or fieldsfile format (or perhaps netCDF ?), probably any GRIB encoding is done in a post-processing step, I suppose this is probably done with Iris.

Are you able to help or can you point me in the direction of someone that can ?

**Comment by BHFock** [comment]
I can point you to post processing systems which perform a mule based field extraction followed by an Iris based GRIB conversion of operational UM output, but currently used fields should be hourly. My understanding is that Iris uses iris-grib (and eccodes) for the GRIB encoding. Details to follow via email.

**Reply by richardgilham** [reply]
Thanks Bjoern!

**Comment by s-boardman** [comment]
We're in touch with Paul D directly for the VIPP systems. I've offered to work out how to send the grib conversions produced by servicehub/mdda for sub-hourly StaGE data.
